packages/repo-app-import-sub-generator/test/prompts/prompts.test.ts

```diff
@@ -258,4 +235,94 @@ describe('getPrompts', () => {
         expect(await (targetFolderPrompt as any)?.when(selectedAnswer)).toBe(true);
         expect(await (targetFolderPrompt as any)?.validate('someother/path', appValue)).toBe(true);
     });
+
+    it('should display GA link when no app is chosen and app list is empty', async () => {
+        const realValidateAppSelection = jest.requireActual('../../src/utils/validators').validateAppSelection;
+        (validateAppSelection as jest.Mock).mockImplementation(realValidateAppSelection);
+
+        const gaLinkObject = {
+            link: {
+                text: 'Need help with this error?',
+                icon: 'data:image/ABC',
+                url: 'https://ga.support.sap.com/dtp/viewer/index.html#/tree/3046/actions/63911'
+            },
+            icon: 'icon/ABC',
+            text: 'Need help with this error?',
+            url: 'https://ga.support.sap.com/dtp/viewer/index.html#/tree/3046/actions/63911',
+            message: 'No applications deployed to this system can be downloaded.'
+        };
+
+        const mockGetHelpLink = ErrorHandler.getHelpLink as jest.Mock;
```

Comment:
Should we test the real GA link instead of mocking, would that be more valuable?